KarhooSDKTests/TestCases/Service/Quote/KarhooCoverageSpec.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  KarhooCoverageSpec.swift
+//  KarhooSDKTests
+//
+//  Created by Nurseda Balcioglu on 10/11/2020.
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import XCTest
+
+@testable import KarhooSDK
+
+final class KarhooCoverageSpec: XCTestCase {
+
+    private var testobject: KarhooQuoteService!
+    private var mockCoverageInteractor = MockCoverageInteractor()
```

Comment:
the mock coverage interactor is for QuoteServiceSpec. (missing test) For this suite of tests you need to mock the request sender.  The testObject should be the KarhooCoverageInteractor, not the quote service. That's in the QuoteServiceSpec class you want to do that.